image/templates/helm/stackrox-central/templates/01-central-08-db-configmap.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     {{- include "srox.annotations" (list . "configmap" "central-db-config") | nindent 4 }}
 data:
   postgresql.conf: |
-    {{- ._rox.central.db._postgresConfig | nindent 4 }}
```

Comment:
This would be an backwards-incompatible change, no?
Users could currently depend on being able to set the postgres config using `central.db.postgresConfig`?

Reply:
Well I was mostly just making sure I could accomplish what I want with this draft.  But I'm glad you mentioned this as this is certainly something we would need to address so both are supported.